src/components/VHeader/VHeaderInternal.vue

```diff
@@ -0,0 +1,137 @@
+          >
+        </li>
+      </ul>
+    </nav>
+    <nav class="flex md:hidden">
+      <VModal
+        :label="$t('header.aria.menu').toString()"
+        variant="full"
+        mode="dark"
+        :visible="isModalVisible"
+        @open="setModalVisibility(true)"
+      >
+        <template #trigger="{ a11yProps }">
+          <VIconButton
+            ref="menuButtonRef"
+            :icon-props="{ iconPath: menuIcon }"
+            :button-props="{ variant: 'plain' }"
+            v-bind="a11yProps"
+            class="border-tx"
+            size="search-medium"
+          />
+        </template>
+
+        <template #top-bar>
+          <div
+            class="flex h-20 w-full justify-between bg-dark-charcoal py-4 text-white pe-3 ps-6"
+          >
+            <VBrand :is-fetching="false" />
+            <VIconButton
+              ref="closeButton"
+              variant="plain"
+              :button-props="{ variant: 'plain' }"
+              :icon-props="{ iconPath: closeIcon }"
+              size="search-medium"
+              class="border-tx text-white"
+              :aria-label="$t('modal.close')"
+              @click="setModalVisibility(false)"
+            />
+          </div>
+        </template>
+        <template #default>
+          <ul class="flex flex-col items-end bg-dark-charcoal text-white">
+            <li v-for="page in allPages" :key="page.id" class="mt-6">
+              <VLink
+                class="heading-5 text-white"
+                :class="currentPage === page.id ? 'font-bold' : ''"
+                :href="page.link"
+                show-external-icon
+                @click="setModalVisibility(false)"
+                >{{ $t(`navigation.${page.id}`) }}</VLink
+              >
+            </li>
+          </ul>
+          <VLink
+            href="https://wordpress.org"
+            class="text-white hover:no-underline"
+          >
+            <i18n
+              tag="p"
+              path="footer.wordpress-affiliation"
+              class="mt-auto flex flex-row items-center text-sm"
+            >
+              <template #wordpress>
+                <WordPress class="aria-hidden text-white" />
+                <span class="sr-only">WordPress</span>
+              </template>
+            </i18n>
+          </VLink>
+        </template>
+      </VModal>
+    </nav>
+  </header>
+</template>
+<script lang="ts">
+import { defineComponent, ref, useRoute, watch } from '@nuxtjs/composition-api'
+
+import usePages from '~/composables/use-pages'
+
+import VIconButton from '~/components/VIconButton/VIconButton.vue'
+import VBrand from '~/components/VBrand/VBrand.vue'
+import VLink from '~/components/VLink.vue'
+import VModal from '~/components/VModal/VModal.vue'
+
+import closeIcon from '~/assets/icons/close.svg'
+import menuIcon from '~/assets/icons/menu.svg'
+import WordPress from '~/assets/wordpress.svg?inline'
+
+export default defineComponent({
+  name: 'VHeaderInternal',
```

Comment:
I don't want to bike shed too much on the name of this component, but I don't really understand what "Internal" means here. Is it only meant for special, non-public pages or something?

Reply:
I think the 'Internal' here comes from the Internal pages that this header links to. The non-search and non-content-details header. Do you have any suggestions for the component name?